File: Class_Member/3.rb
class Cs
  @@count = 0 # 변수 초기화 
  def initialize() 
    @@count += 1   # @ = instance변수, @@ = Class변수
  end
  def Cs.getCount()
    return @@count
  end
end

i1 = Cs.new()
i2 = Cs.new()
i3 = Cs.new()
i4 = Cs.new()
p Cs.getCount() # Output = 4

File: Class_Member/3.py
class Cs:
  count = 0 # 초기화
  def __init__(self):
    Cs.count += 1
  @classmethod
  def getCount(cls):
    print(cls)
    return Cs.count# = cls.count (cls = Cs)

i1 = Cs()
i2 = Cs()
i3 = Cs()
i4 = Cs()
print(Cs.getCount())